embassy-time/src/timer.rs

```diff
@@ -190,6 +190,18 @@ impl Ticker {
         self.expires_at = Instant::now() + self.duration;
     }
 
+    /// Reset the ticker to fire for the next time on the deadline.
+    /// If the deadline is in the past, the ticker will fire instantly.
+    pub fn reset_at(&mut self, deadline: Instant) {
+        self.expires_at = deadline;
```

Comment:
this is inconsistent with `reset` and `reset_after`. 

`reset` resets it at "now", so it expires next at "now + self.duration"
`reset_after` resets it at "now + after", so it expires next at "now + after + self.duration"

so if `reset_at` resets it at "deadline", it should expire next at "deadline + self.duration"

Reply:
I've updated it accordingly.

Reply:
can't see the change, maybe you didn't push?

Reply:
I pushed it to the wrong branch. After fighting with git for a while, it should've worked now.

Reply:
I pushed it to the wrong branch. After fighting with git for a while, it should've worked now.